README.md

```diff
@@ -2,12 +2,20 @@
 
 Autodesk Filmbox (*.fbx) loader for Bevy Engine.
 
+<!-- TODO: ### Features -->

```

Comment:
```suggestion
### Features

Current features:

- Loads meshes, textures and some materials properties (including diffuse texture, normal maps and occlusion maps)
- Supports Maya PBR material
- Supports scene tree transform hierarchy

**Planned** features:

- skeleton rig imports
- `bevy_animation` support
- An optional Phong/Lambert shader implementation
  for the most common type of FBX materials
- More compatibility (`IndexToDirect`, handle file axis properties,
  cameras and lights, N-gon triangulation)
```

Reply:
Phong is not a material, it's interpolation technique, shading model & reflection model used for PBR/BRDF.